BundleClient/app/src/main/java/com/ddd/wifidirect/WifiDirectManager.java

```diff
@@ -364,6 +373,14 @@ public String getGroupHostIP() {
      */
     public Context getContext() { return this.context; }
 
+    @Override
+    public void onPeersAvailable(WifiP2pDeviceList peers) {
+        for (WifiP2pDevice device : peers.getDeviceList()) {
+            discoveredPeers.add(device);
```

Comment:
this seems wrong in the long run. above, we made a new list so peers would fall off the list.